modules/tinymce/src/core/main/ts/delete/TableDelete.ts

```diff
@@ -50,13 +50,28 @@ const handleEmptyBlock = (editor: Editor, startInTable: boolean, emptyBlock: Opt
   });
 };
 
-const setCursorInCell = (editor: Editor, cell: SugarElement<HTMLTableCellElement>) => {
+const deleteContentInsideCell = (cell: SugarElement<HTMLTableCellElement>, rng: Range, isFirstCellInSelection: boolean) => {
+  const insideTableRng = rng.cloneRange();
+
+  if (isFirstCellInSelection) {
+    insideTableRng.setStart(rng.startContainer, rng.startOffset);
+    insideTableRng.setEndAfter(cell.dom.lastChild);
+  } else {
+    insideTableRng.setStart(cell.dom.firstChild, 0);
```

Comment:
```suggestion
    insideTableRng.setStartBefore(cell.dom.firstChild);
```

I feel like these two branches should mirror each other. However, it _might_ not make a difference in terms of functionality once you start deleting things?